### ISSUE TITLE
[build-release] Import multiple stage build

### ISSUE THREAD

**Issue by gaocegege** [sjtug/lug#62]
Build the binary in image 1 and copy it into the image 2, and the image2 is for prod, WDYT @htfy96 

Pros:

- Reduce size
- Easy to maintain, do not rely on the wrapper

**Comment by gaocegege** [comment]
#dibs

**Comment by htfy96** [comment]
LGTM. And we'd better keep the final image debian-based, since [`mirror-docker`](https://github.com/sjtug/mirror-docker/blob/v2/lug/Dockerfile) relies on this image and assumes it is Debian.

**Comment by gaocegege** [comment]
```dockerfile
FROM golang:alpine AS build-env
# The GOPATH in the image is /go.
ADD . /go/src/github.com/sjtug/lug
WORKDIR /go/src/github.com/sjtug/lug
RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
RUN dep ensure
RUN go build github.com/sjtug/lug/cli/lug

FROM alpine:3.7
WORKDIR /app
COPY ./config.example.yaml /app/
COPY --from=build-env /go/src/github.com/sjtug/lug/lug /app/
ENTRYPOINT ["./lug"]
```

Does it work for you? 我不知道 `COPY ./config.example.yaml /app/` 是不是必要的，服务器上怎么跑起来的呢

**Comment by gaocegege** [comment]
OK cool, I will change alpine to debian or ubuntu.

**Comment by htfy96** [comment]
`config.example.yaml`是mount进去的

**Comment by htfy96** [comment]
Fixed in #63
